iron-fit-behavior.html

```diff
@@ -111,7 +179,40 @@
       return fitTop;
     },
 
+    /**
+     * The element that should be used to position the element when
+     * it opens, if no position target is configured.
+     */
+    get _defaultPositionTarget() {
+      var parent = Polymer.dom(this).parentNode;
+
+      if (parent.nodeType === Node.DOCUMENT_FRAGMENT_NODE) {
+        parent = parent.host;
+      }
+
+      return parent;
+    },
+
+    /**
+     * The horizontal align value, accounting for the RTL/LTR text direction.
+     */
+    get _localeHorizontalAlign() {
+      if (this._isRTL) {
+        // In RTL, "left" becomes "right".
```

Comment:
I think this flipping is a little counter-intuitive.  `horizontal-align="{start,end}"` would make more sense for auto-flippy values and might not break others like this did for [Chrome's downloads UI](https://bugs.chromium.org/p/chromium/issues/detail?id=640058), which was [detecting RTL itself](https://codereview.chromium.org/1385113002/diff/1/chrome/browser/resources/md_downloads/toolbar.js?context=10&column_width=80&tab_spaces=8) before this change landed :(.